Source/WebKit/Platform/cocoa/WebPrivacyHelpers.mm

```diff
@@ -414,6 +426,89 @@ void requestLinkDecorationFilteringData(LinkFilteringRulesCallback&& callback)
     }];
 }
 
+static uint64_t approximateContinuousTimeNanoseconds()
+{
+    return clock_gettime_nsec_np(CLOCK_MONOTONIC_RAW_APPROX);
```

Comment:
I just wanted an approximate clock that incremented while the system was sleeping, which is `CLOCK_MONOTONIC_RAW_APPROX` or `mach_continuous_approximate_time`, but it looks like we don't have wrappers around those in WebKit right now.

Reply:
I see, this seems fine then.